library/mobjects.c

```diff
@@ -95,7 +95,7 @@ int eblob_base_setup_data(struct eblob_base_ctl *ctl, int force)
 		goto err_out_exit;
 	}
 
-	if ((st.st_size && ((unsigned long long)st.st_size != ctl->data_ctl.size)) || force) {
+	if (st.st_size && ((unsigned long long)st.st_size != ctl->data_ctl.size)) {
```

Comment:
I don't agree with any condition here. The next line should be executed without any condition (behavior like `force == true`)

Reply:
agree

Reply:
@sabramkin, take a look please.